environments/overlays/cicd/pipeline-bot.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Maybe we can change the file name to something more meaningful so users can understand what it is.
WDYT about something like `pipeline-credentials.yaml`

Reply:
Done. Needs to rebase before merge.